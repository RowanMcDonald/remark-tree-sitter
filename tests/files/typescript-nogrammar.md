```typescript
function foo() {
  return 1;
}
```