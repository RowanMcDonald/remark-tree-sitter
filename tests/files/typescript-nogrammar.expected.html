<pre><code class="language-typescript">function foo() {
  return 1;
}
</code></pre>
